contracts/protocol/bases/SellerBase.sol

```diff
@@ -92,6 +92,16 @@ contract SellerBase is ProtocolBase, IBosonAccountEvents {
         _seller.id = sellerId;
         storeSeller(_seller, _authToken, lookups);
 
+        // Set treasury as the default royalty recipient
+        if (_voucherInitValues.royaltyPercentage > protocolLimits().maxRoyaltyPercentage)
+            revert InvalidRoyaltyPercentage();
+        RoyaltyRecipient[] storage royaltyRecipients = lookups.royaltyRecipientsBySeller[sellerId];
+        RoyaltyRecipient storage defaultRoyaltyRecipient = royaltyRecipients.push();
+        defaultRoyaltyRecipient.wallet = _seller.treasury;
+        defaultRoyaltyRecipient.minRoyaltyPercentage = _voucherInitValues.royaltyPercentage;
```

Comment:
What happen if we set different values for royaltyPercentage per collection?
Shouldn't we apply the default percentage to the offer, regarding the collection it belongs to ?

Reply:
With this PR, the royalty management happens on two levels:
- The seller's admin sets a list of allowed royalty recipients and the minimal royalty percentage they should get. That's done relatively infrequently.
- The seller's assistant picks the actual royalty recipients for each offer individually. They must be chosen from the list of allowed recipients. The assistant can also choose that nobody gets any royalties.

In this place in the code, we are in the place where the treasury is added to the list of allowed recipients. Treasury will always be on this list and cannot be removed. It's called "default" since it's always on this list, although this does not mean that when the offer is created, it is by default one of the recipients. Assistant must actively add it to the offer and they can also decide that the treasury is not getting any royalties.